src/Translumo.TTS/Engines/SileroTTSEngine.cs

```diff
@@ -146,5 +148,9 @@ lang switch
             _ => null
         };
 
+    public string[] GetVoices() => _voices;
+
+    public void SetVoice(string voice) => _voice = _voices.First(x => x.Equals(voice, StringComparison.OrdinalIgnoreCase));
```

Comment:
There is a bug, exception occurred when TTS with voice is selected, then user switch to another TTS. Need to set TtsConfiguration.CurrentVoice after(before) this update.